.github/workflows/main-branch-deployment.yml

```diff
@@ -0,0 +1,51 @@
+name: Main Branch Deployment
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
+      - '[0-9]+-**'
+    paths-ignore:
+      - '**.drawio'
+      - '**.md'
+      - '**.png'
+  pull_request:
```

Comment:
Is the pull request trigger necessary? I think the on.push.main is enough.

Reply:
Looks like it's not needed. Good catch!